R/slice.R

```diff
@@ -303,7 +303,20 @@ slice_eval <- function(mask, dots, error_call = caller_env()) {
 
     for (i in seq_len(n)) {
       index <<- i
-      out[[i]] <- ...elt2(i)
+
+      slice_idx <- ...elt2(i)
+      if (is.matrix(slice_idx) && ncol(slice_idx) == 1) {
+        lifecycle::deprecate_warn("1.1.0", I("Slicing with a 1-column matrix"))
+        slice_idx <- slice_idx[, 1]
+      }
+
+      out[[i]] <- vec_as_subscript(
+        slice_idx,
+        logical = "error",
+        character = "error",
+        arg = as_label(dots[[i]]),
+        call = NULL # error always chained to select()
```

Comment:
```suggestion
        call = NULL # error always chained to slice()
```

Was it select or slice?

Reply:
Ooops